Fix eslint issues

```bash
/var/www/html/web/modules/contrib/localgov_alert_banner/js/alert_banner.js
  41:30  warning  Unexpected unnamed function  func-names
  57:41  warning  Unexpected unnamed function  func-names
  58:49  warning  Unexpected unnamed function  func-names
```

--- a/js/alert_banner.js
+++ b/js/alert_banner.js
@@ -38,7 +38,7 @@ function setAlertBannerHideCookie(cookieTokens, token) {
       );
 
       if (alertBanners) {
-        alertBanners.forEach(function (banner) {
+        alertBanners.forEach((banner) => {
           banner.classList.remove('hidden');
           const token = banner.getAttribute('data-dismiss-alert-token');
           if (cookieTokens.includes(token)) {
@@ -54,8 +54,8 @@ function setAlertBannerHideCookie(cookieTokens, token) {
       );
 
       if (alertBannerCloseButtons) {
-        alertBannerCloseButtons.forEach(function (closeButton) {
-          closeButton.addEventListener('click', function (e) {
+        alertBannerCloseButtons.forEach((closeButton) => {
+          closeButton.addEventListener('click', (e) => {
             e.preventDefault();
             const banner = closeButton.closest('.js-localgov-alert-banner');
             banner.setAttribute('aria-hidden', 'true');

